frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -221,18 +221,18 @@ function extractPersonData(row: SurveyResponseRow): {
 } {
     const distinctId = row.at(-2) as string
     const timestamp = row.at(-1) as string
-    const unparsedPersonProperties = row.at(-3)
-    let personProperties: Record<string, any> | undefined
-
-    if (unparsedPersonProperties && unparsedPersonProperties !== null) {
-        try {
-            personProperties = JSON.parse(unparsedPersonProperties as string)
-        } catch {
-            // Ignore parsing errors for person properties
+    // now, we're querying for all PERSON_DEFAULT_DISPLAY_NAME_PROPERTIES, starting from the third last value, so build our person properties object
+    // from those values. We use them to have a display name for the person
+    const personProperties: Record<string, any> = {}
+    let hasAnyProperties = false
+    for (let i = 0; i < PERSON_DEFAULT_DISPLAY_NAME_PROPERTIES.length; i++) {
+        const value = row.at(-3 - i) as string
+        personProperties[PERSON_DEFAULT_DISPLAY_NAME_PROPERTIES[i]] = value
+        if (value && value !== null && value !== '') {
+            hasAnyProperties = true
```

Comment:
```suggestion
        if (value && value !== null && value !== '') {
            personProperties[PERSON_DEFAULT_DISPLAY_NAME_PROPERTIES[i]] = value
            hasAnyProperties = true
```

Reply:
otherwise you might be setting null/empty values to `personProperties`, and we dont want that?